workflows/sarek.nf

```diff
@@ -234,6 +241,11 @@ workflow SAREK {
     pon_tbi                = params.pon                     ? params.pon_tbi               ? Channel.fromPath(params.pon_tbi).collect()               : PREPARE_GENOME.out.pon_tbi               : []
     msisensorpro_scan      = PREPARE_GENOME.out.msisensorpro_scan
 
+    // Gather index for mapping given the chosen aligner
+    ch_map_index = params.aligner == "bwa-mem" ? bwa :
+        params.aligner == "bwa-mem2" ? bwamem2 :
+        dragmap
+
     //TODO @Rike, is this working for you? Now it is, fixed a bug in prepare_genome.nf after chasing smoke for a while
```

Comment:
This line can be removed. also pretty sure we discussed this again in a repvious PR